packages/ibm-products-styles/src/components/SidePanel/_side-panel.scss

```diff
@@ -139,15 +129,8 @@ $action-set-block-class: #{c4p-settings.$pkg-prefix}--action-set;
       margin-bottom: 0;
     }
     &.#{$block-class}__on-detail-step .#{$block-class}__collapsed-title-text {
-      top: $spacing-07;
+      top: $spacing-09;
```

Comment:
Any specific reason we increased this by `1rem`? Seems minor but does also increase the vertical real estate used by the header. 🤔